Second-round copyedit on drivers.tex (#72)
CHERIoT-Platform/book: text/drivers.tex

\begin{description}
	\item[tag=Abstraction]{ A device driver allows software to be written to interface with a \keyword{device class} rather than a single device.}

> 	\item[tag=Multiplexing]{ A device may need to be accessed by different software components.}
> \amanda{I recommend changing the description of multiplexing to read something like "A device driver allows multiple different software components to access the device." so that it's more consistent with the description for Abstraction.}

Reviewer: ```suggestion
	\item[tag=Multiplexing]{A device driver allows multiple compartments to share access to a device without needing to trust each other.}
```

Author: Updated and comment removed.

The macro expands to a load from the compartment's import table.
Assigning the result of it to a global is an antipattern: you will get smaller code using it directly.
The helper shown here will be inlined and expand to a single load capability load.

> \amanda{Should "load" be in this sentence twice?}

Reviewer: No, though I'm not sure which is better to delete (either is probably correct).

Author: I picked one and removed it.
